### ISSUE TITLE
About File/Image fields on model-driven apps

### ISSUE THREAD

**Issue by phyle** [MicrosoftDocs/powerapps-docs#1477]
In section "File fields", it mentions "Currently, the file data type is only available to canvas apps and flows."  However this must have recently changed has it is now available from model-driven apps too, only via the new Forms designer. (Power Apps Make > Solution > [selection solution] > [select an entity like Contact] > Fields tab > Add field (command) > select "File" in Data type)

In the designer, there's no hint that it is a pre-release preview (on 2020 release wave 1)... Is the mention "[This topic is pre-release documentation and is subject to change.]" still true?

Also, while they are fields, are File and Image fields stored in the "Database" or the "File" storage in the capacity model?

Thanks

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6d193924-c732-6c92-721b-348baede9d54
* Version Independent ID: c871720f-64d4-66e7-7b6d-1c555e4e32e1
* Content: [Field data types in Common Data Service - Power Apps](https://docs.microsoft.com/en-us/powerapps/maker/common-data-service/types-of-fields#file-fields)
* Content Source: [powerapps-docs/maker/common-data-service/types-of-fields.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/live/powerapps-docs/maker/common-data-service/types-of-fields.md)
* Service: **powerapps**
* GitHub Login: @Mattp123
* Microsoft Alias: **matp**

**Comment by Mattp123** [comment]
@phyle - thank you for your feedback. You are correct that the File data type is now also supported with model-driven apps. I've updated the topic to reflect this, which will be live soon. The File data type feature however, is still in public preview. 

I'm trying to get an answer for you question about where the data is stored.

**Comment by Mattp123** [comment]
@phyle - File and image data is measured against capacity under File and database. More information: https://docs.microsoft.com/en-us/power-platform/admin/capacity-storage#capacity-page-details

Closing issue.

**Comment by phyle** [comment]
Capacity Storage is divided in 3 types: Database, File, Log.  I don't understand your answer, the file/image data can't (shouldn't) be stored in both File and Database at the same time.  Unless in the Database type, there's a ID/URL to the actual blob storage in the File storage type?

**Comment by mayadumesh** [comment]
I would like to demystify proceedings for capacity storage as Database (relational datastore, indexes and cross -reference of data), File ( immutable binaries stored within the system) and Log (CRD data that can not be updated).

Metadata about File field (name, size, and so on) is part of database storage however binary payload is File storage.

**Comment by phyle** [comment]
Thanks Matt for the document update and thanks Maya for the precisions!
